guides/getting-started.md

```diff
@@ -5,29 +5,47 @@ For existing apps, follow these steps to begin using Angular Material.
 
 ### Step 1: Install Angular Material and Angular CDK
 
+#### NPM
 ```bash
 npm install --save @angular/material @angular/cdk
 ```
+#### Yarn
+```bash
+yarn add @angular/material @angular/cdk
+```
+
```

Comment:
I think this needs something that clearly conveys it would be one or the other, not both.

Reply:
@jelbourn something like `We have added install scripts so you can pick between **npm** or **yarn**`

Reply:
Right under "Step 1", it should say 

"You can use either the `npm` or `yarn` command-line tool to install packages. Use whichever is appropriate for your project in the examples below."

Reply:
@jelbourn added it 👍